src/Bridges/bridgeoptimizer.jl

```diff
@@ -239,12 +239,25 @@ end
 function MOI.supports(b::AbstractBridgeOptimizer,
                       attr::MOI.AbstractConstraintAttribute,
                       IndexType::Type{<:MOI.Index})
-    return MOI.supports(b.model, attr, IndexType)
+    if is_bridged(b, IndexType)
+        return MOI.supports(b.bridged, attr, IndexType)
```

Comment:
This is only if `attr isa MOI.ConstraintName`, otherwise, it should be `MOI.supports(b, attr, concrete_bridge_type(b, F, S))`